src/test_l1_dsFPD.c

```diff
@@ -123,6 +127,18 @@ static int gTestID = 1;
     }\
 }\
 
```

Comment:
Remove DS_ASSERT_AUTO_TERM_NUMERICAL and DS_ASSERT_AUTO_TERM_STRING if not used.